### ISSUE TITLE
 data processing

### ISSUE THREAD

**Issue by zhao706** [mrvturan96/Brain-Tumor-Detection-and-Segmentation-using-Deep-Learning#1]
Hello,are you using the via tool to mark comments that manually label the images in the t1ce folder?

**Comment by mrvturan96** [comment]
Hi,
Yes, I used via tool for annotations and labeled manually.



zhao706 <notifications@github.com>, 1 Mar 2020 Paz, 12:53 tarihinde şunu
yazdı:

**Comment by zhao706** [comment]
Thank you. But if you mark the location of brain tumor manually, the final segmentation results will be affected by the accuracy of people's labeling. @mrvturan96

**Comment by mrvturan96** [comment]
You're right. Actually, we labeled only t1ce manually for Mask-RCNN. For this, we convert NIFTI images to .jpg images. Tumors very pronounced on t1ce sequence. (We used only the most obvious slide.) 
But we didn't manually labeled on U-Net. Because NIFTI images labeled **by radiologists.**

**Comment by zhao706** [comment]
OK, I see.Thank you very much.




------------------ Original ------------------

**Comment by javaidiqbal11** [comment]
Hello, 
Thanks for sharing this work, I want some help to execute the code and also worked on the same project to add some work with you. Please reply to me soon.
jtechofficial198@gmail.com

**Comment by javaidiqbal11** [comment]
> OK, I see.Thank you very much.
> […](#)
> ------------------ Original ------------------ From: Merve TURAN <notifications@github.com&gt; Date: Wed,Mar 4,2020 2:26 PM To: mrvturan96/Brain-Tumor-Detection-and-Segmentation-using-Deep-Learning <Brain-Tumor-Detection-and-Segmentation-using-Deep-Learning@noreply.github.com&gt; Cc: zhao706 <1043314579@qq.com&gt;, Author <author@noreply.github.com&gt; Subject: Re: [mrvturan96/Brain-Tumor-Detection-and-Segmentation-using-Deep-Learning] data processing (#1) You're right. Actually, we labeled only t1ce manually for Mask-RCNN. For this, we convert NIFTI images to .jpg images. Tumors very pronounced on t1ce sequence. (We used only the most obvious slide.) But we didn't manually labeled on U-Net. Because NIFTI images labeled by radiologists. — You are receiving this because you authored the thread. Reply to this email directly, view it on GitHub, or unsubscribe.

Can you guide me on what steps you follow to execute the complete work now. please

**Comment by javaidiqbal11** [comment]
What the research methodology you proposed and can you please share the research paper for this work.
